gpkit/nomials/nomial_math.py

```diff
@@ -469,6 +451,25 @@ def __init__(self, left, oper, right):
         self.nomials = [self.left, self.right, self.p_lt, self.m_gt]
         self.nomials.extend(self._unsubbed)
 
+    def simplify_posy_ineq(self, exps, cs):
+        "Simplify a posy <= 1 by moving constants to the right side."
```

Comment:
I find it confusing that this is a method of `self` but also takes `exps` and `cs` as arguments. I see that it was needed for printing, but, can exps and cs be anything here? Or do they need to relate to self in some way?